### ISSUE TITLE
graphic glitches in mermaid expression

### ISSUE THREAD

**Issue by grkngls** [careteditor/caret#584]
following md **mermaid** code looks not at it should:

```mermaid
graph TD
a[Start] --> b{"this is just a test"}
b --> |one| c[This is a longer text which escape the box]
b --> |two| d["Another text<br>with l a line break.]
```

[
![dqwvrdsuiae4c1k](https://user-images.githubusercontent.com/11460671/33984031-82c579fe-e0b6-11e7-8352-73557ebd7997.jpg)
](url)

BTW: not so easy to enter the md code here as raw code.

**Comment by astoilkov** [comment]
We will look into this. Thanks for reporting.

**Comment by cboisen** [comment]
Which release has this been resolved in? @astoilkov

**Comment by astoilkov** [comment]
Hi @hauthorn,

We have removed mermaid support in Caret. This is why we added the `resolved` label.

**Comment by tikkanz** [comment]
Is there a plan to reinstate support for mermaid at some point or is the removal permanent?

**Comment by astoilkov** [comment]
Yes. We have plans to support a diagramming library at some point in the future. However, we can't promise anything at this point.

**Comment by cboisen** [comment]
@astoilkov You've probably already looked at this, but is it possible that the custom fonts used are causing the labels to render wrongly? I recent came about this in the [documentation for mermaid](https://mermaidjs.github.io/usage.html#labels-out-of-bounds)

**Comment by astoilkov** [comment]
@hauthorn Yes. It is possible. Thanks for pointing this out. We will look at this.
